.github/workflows/build-package-docs.yaml

```diff
@@ -0,0 +1,95 @@
+name: Ansible package docs build
+on:
+  workflow_dispatch:
+    inputs:
+      github_fork:
+        description: 'GitHub Fork'
+        required: true
+        default: 'ansible'
+      github_branch:
+        description: 'GitHub Branch'
+        required: true
+        default: 'devel'
+      language:
+        type: choice
+        description: 'Language'
+        required: true
+        default: 'english'
+        options:
+        - 'english'
+        - 'japanese'
+      package_version:
+        type: choice
+        description: 'Ansible Version'
+        required: true
+        default: 'devel'
+        options:
+        - 'devel'
+        - '10'
+        - '9'
+        - '8'
+        - '7'
+      latest_symlink:
+        type: boolean
+        description: 'Add latest symlink'
+        required: true
+
+jobs:
+  build-package-docs:
+    runs-on: ubuntu-latest
+    env:
+      PACKAGE_VERSION: "${{ github.event.inputs.package_version }}"
+      LANGUAGE: ${{ github.event.inputs.language}}
+    steps:
+      - name: Checkout Ansible documentation
```

Comment:
This is inconsistent with the indentation above. I'd dedent the list of steps. Here's the yamllint config that I like to use, in case you'd like to integrate it at some point: https://github.com/aio-libs/yarl/blob/bd5ff24/.yamllint.